src/map.html

```diff
@@ -23,141 +23,116 @@
-        <div class="panel panel-default">
-          <div class="panel-heading">
-<!--             <h4 class="panel-title"><a data-toggle="collapse" data-parent="#accordion" href="#collapse1">Select Industry</a></h4> -->
-            <h4 class="panel-title"><a data-toggle="collapse" href="#collapse1">Select Industry</a></h4>
-          </div>
-          <div id="collapse1" class="panel-collapse collapse">
-            <ul class="list-group">
-              <li class="list-group-item"><input type="checkbox" value=""> Industry 1</li>
-              <li class="list-group-item"><input type="checkbox" value=""> Industry 2</li>
-            </ul>
+      <br><br><br>
+
+  <!-- Select Industry -->
+      <div class="col-xs-6">
+        <div class="panel-group" id="accordion">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h4 class="panel-title"><a data-toggle="collapse" href="#collapse1">Select Industry</a></h4>
+            </div>
+            <div id="collapse1" class="panel-collapse collapse">
+              <ul class="list-group"></ul>
+            </div>
           </div>
         </div>
       </div>
-    </div>
 
-<!-- Select Features -->
-    <div class="col-xs-6">
-      <div class="panel-group" id="accordion">
-        <div class="panel panel-default">
-          <div class="panel-heading">
-            <h4 class="panel-title"><a data-toggle="collapse" href="#collapse2">Select Feature</a></h4>
-          </div>
-          <div id="collapse2" class="panel-collapse collapse">
-            <ul class="list-group">
-              <li class="list-group-item"><input type="checkbox" value=""> Feature 1</li>
-              <li class="list-group-item"><input type="checkbox" value=""> Feature 2</li>
-            </ul>
+  <!-- Select Features -->
+      <div class="col-xs-6">
+        <div class="panel-group" id="accordion">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h4 class="panel-title"><a data-toggle="collapse" href="#collapse2">Select Feature</a></h4>
+            </div>
+            <div id="collapse2" class="panel-collapse collapse">
+              <ul class="list-group"></ul>
+            </div>
           </div>
         </div>
       </div>
-    </div>
-
-<br><br><br>
 
-<!--     <div class="col-xs-12">
-      <button type="button" class="btn btn-default btn-lg" style="text-align:center">Draw Regions</button>
-    </div> -->
+  <br><br><br>
 
-  </div>
-
-<!--  Map -->
+    </div>
 
-  <div class="col-sm-8">
-    <h3>Map Display</h3><br>
-    <div id="mapid">
-  </div>
+  <!--  Map -->
 
-  <!-- Displaying the map (in JS) -->
-  <script type="text/javascript" src="public/map.js">
+    <div class="col-sm-8">
+      <h3>Map Display</h3>
+      <br>
+      <div id="mapid">
+    </div>
 
-  </script>
+    <!-- Displaying the map (in JS) -->
+    <script type="text/javascript" src="public/map.js"></script>
 
-</div></div>
+  </div></div>
```

Comment:
Could you split this onto two lines, please? Easier to keep track of indentation.